src/utils/datetime.js

```diff
@@ -110,3 +110,18 @@ export function millisecondsAsTimeOfDay(milliseconds, format = 'h:mm a') {
   }
   return moment.utc(milliseconds).format(format);
 }
+
+/**
+ * formatDisplayDate
+ * @param  {Number} timestamp integer hammertime
+ * @param  {Object} timePrefs object containing timezoneAware Boolean and timezoneName String
+ *
+ * @return {String}           formatted timezoneAware date string
+ */
+export function formatDisplayDate(timestamp, timePrefs) {
+  if (!timestamp) {
+    return '';
+  }
+  return moment(timestamp).utc().tz(getTimezoneFromTimePrefs(timePrefs))
```

Comment:
So the issue here is that `moment().utc()` is not the same as `moment.utc()` - what I've been using is the latter, and I think my reasons for doing so are sound. I've documented here as a starting point for discussion and (hopefully) convergence: http://momentjs.com/docs/#/parsing/utc/